### ISSUE TITLE
Creating a 1D solver input file fails processing the rcrt.dat file

### ISSUE THREAD

**Issue by ktbolt** [SimVascular/SimVascular#769]
When going through the 1D Simulation **Demo Project** tutorial clicking **Create Files for Simulation** generates the following error
```
[generate-1d-mesh] INFO - Set variable outflow boundary conditions ...
[generate-1d-mesh] ERROR - Error while reading variable outflow boundary conditions
[generate-1d-mesh] ERROR - 'r'
```

This error is caused by the `read_variable_outflow_bcs()` function is creating the rcrt.dat file name using 
```
bc_file = path.join(params.outflow_bc_file, outflow_bc_sv)
```
which appends a directory named **r** to the file name: **Simulations1d/demo/rcrt.dat/r**.

**Comment by mrp089** [comment]
Thanks for pointing this out @ktbolt! Fixed in #770.

The bug was introduced in #624 by accepting only the "new" format for boundary conditions (see #740). The code can now handle both "old" and "new".

**Comment by ktbolt** [comment]
@mrp089 Quick fix!

The problem was found by a user using the SV beta version. I instructed them to use the SV 2020-04 release which works fine.
